src/site/content/en/blog/codelab-signin-form/index.md

```diff
@@ -30,89 +32,134 @@ As you'll see, these elements enable built-in browser functionality, improve acc
 Add the following code inside the `<body>` element:
 
 ```html
-<form id="signin" action="#" method="post">
-
+<form id="login" action="#" method="post">
   <h1>Sign in</h1>
-
   <section>
     <label>Email</label>
     <input />
   </section>
-
   <section>
     <label>Password</label>
     <input />
   </section>
-
   <button>Sign in</button>
-
 </form>
 ```
-{% Aside 'codelab' %}
-[View complete code for Step 1](https://glitch.com/edit/#!/signin-form-codelab-1).
-{% endAside %}
+
+Here's how your `index.html` should look at this point:
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/login-form-best-practices-codelab-step-1?path=index.html&attributionHidden=true&sidebarCollapsed=true"
+    alt="How the Glitch should look after completing step 1."
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+Click **View App** to preview your login form. 
+The HTML you just added is valid and correct, but the default browser styling
+means it looks terrible and it's hard to use, especially on mobile.
```

Comment:
A key thing that I want to see throughout the codelab is more explicit interactivity. We should tell the reader when to look at the app and what to notice.

Reply:
👍